specs/windows/shimcache.table

```diff
@@ -0,0 +1,12 @@
+table_name("shimcache")
+description("Application Compatibility Cache, contains artifacts of execution.")
+schema([
+    Column("entry", INTEGER, "Execution order."),
+    Column("path", TEXT, "This is the path to the executed file."),
+    Column("modified_time", INTEGER, "File Modified time."),
+    Column("execution_flag", TEXT, "Boolean Execution flag (Flag does not exist on Windows 10 and higher)."),
```

Comment:
This looks great!

One last consideration for consistency. In a lot of other tables we represent "tri-states" in columns as either {-1, 0, 1} or {NULL, 0, 1}. My recommendation is to follow that pattern for `execution_flag` in the schema.

So concretely:
- Change this line to be an `INTEGER` type.
- Update the description to be one of either: "Boolean Execution flag, 1 for execution enabled, 0 for disabled, and NULL for missing (this flag does not exist on Windows 10 and higher)." or substitute "NULL" for "-1" depending on what you choose.
- Update where the column is set to swap `true` for `1` and `false` for `0`.

Reply:
Changed to `INTEGER` type, updated description, and switched column to to 1,0, or -1
Also just to clarify the execution flag specifies whether the file was executed (1 - executed, 0 - not executed, -1 - flag is missing)